File: pgm/ABC_Visual_Basic/ABC_401_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As Integer
        S = Console.ReadLine()
        Const lower As Integer = 200
        Const upper As Integer = 299
        Const Success As String = "Success"
        Const Failure As String = "Failure"
        If lower <= S AndAlso S <= upper Then
            Console.WriteLine(Success)
        Else
            Console.WriteLine(Failure)
        End If
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_401_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        int S = int.Parse(Console.ReadLine());
        const string Success = "Success";
        const string Failure = "Failure";
        const int twoHundred = 200;
        const int twoHundredNinetyNine = 299;
        Console.WriteLine(
            twoHundred <= S && S <= twoHundredNinetyNine
                ? Success
                : Failure);
        Environment.Exit(0);
    }
}